packages/p2p/src/pages/app.jsx

```diff
@@ -17,7 +17,10 @@ import Routes from 'Components/routes';
 import './app.scss';
 
 const App = () => {
-    const { is_p2p_v2_enabled } = useFeatureFlags();
+    const [is_p2p_v2_enabled, isGBLoaded] = useGrowthbookGetFeatureValue({
```

Comment:
Can you help to rename p2p_v2 to p2p_standalone and isGBLoaded to isGrowthbookLoaded for them to be more descriptive.

Reply:
done